lib/Subscription/Blacklist.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace MailPoet\Subscription;
+
+class Blacklist {
+  const SALT = 'mailpoet';
+
+  private $blacklist = [
+    'e60c6e0e73997c92d4ceac78a6b6cbbe6249244c4106a3c31de421fc50370ecd' => 1,
+  ];
+
+  public function isBlacklisted($email) {
+    $hashed_email = $this->hash($email);
+    return !empty($this->blacklist[$hashed_email]);
+  }
+
+  public function hash($email) {
+    return hash('sha256', $email . self::SALT);
+  }
+
+  public function addEmail($email) {
```

Comment:
Removed this method. Will pass data in constructor for tests because `hash()` is now private and I don't want to duplicate the hashing routine in tests.